profiles/ug/themes/ug/ug_theme/js/ug_scripts.js

```diff
@@ -1,53 +1,12 @@
-/*************************
-Dropdown Menus
-*************************/
-
-	/*---------------------
-	Expanded/Collapsed State Fix
-	---------------------**/
-	jQuery(
-	  	function($) {
-
-		// On dropdown open
-		$(document).on('shown.bs.dropdown', function(event) {
-		  var dropdown = $(event.target);
-		 
-		  // Set aria-expanded to true
-		  // dropdown.find('.dropdown-menu').attr('aria-expanded', true);
-		  // dropdown.find('.dropdown-toggle .toggle-indicator').html('hide menu ');
-		 
-		  // Set focus on the first link in the dropdown
-		  setTimeout(function() {
-		    dropdown.find('.dropdown-menu li:first-child a').focus();
-		  }, 10);
+/*-------------------------------------
+Bootstrap collapsible aria-expanded fix
+------------------------------------**/
+jQuery(function($) {
+	$(document).ready(function(e) {
+		$(".collapse").each(function(i, elem) {
+			var isExpanded = $(elem).hasClass('in');
+			$(elem).attr("aria-expanded", isExpanded);
+			$('[data-target="#' + $(elem).attr("id") + '"]').attr("aria-expanded", isExpanded);
```

Comment:
According to the Bootstrap 4 docs, data-target is required for both elements (https://v4-alpha.getbootstrap.com/components/collapse/). We are using an older version or Bootstrap, but it doesn't hurt to have the `data-target` attribute on a link element and will make selectors easier to write.